src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,66 @@
+import classNames from 'classnames';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  filterStatus: FilterStatus
+  onFilter: (filterStatus: FilterStatus) => void;
+};
+
+export const Filter: React.FC<Props> = ({ filterStatus, onFilter }) => {
+  const handleFilter = (newFilterStatus: FilterStatus) => {
+    if (filterStatus !== newFilterStatus) {
+      onFilter(newFilterStatus);
+    }
+  };
+
+  const handleFilterAll = () => {
+    handleFilter('all');
```

Comment:
Do not rely on the unknown string, make constants for this.
```
const FILTERS = {
  all: ‘all’,
  completed: ‘completed’,
  active: ‘active’,
};
```

Reply:
Fixed. Now enum values are used